compose/cli-command.md

```diff
@@ -72,6 +72,12 @@ $ docker-compose disable-v2
 
 ### Install on Linux
 
+#### Installation via per distro packages
+
+You can install Compose V2 using the [per distro repositories](https://docs.docker.com/engine/install/#server) via the `docker-compose-plugin` package.
```

Comment:
@glours Wondering if you are already making changes similar to this as part of you Compose v2 docs? Thanks